src/NetworkTestWidget/EdgeResult/EdgeResult.test.tsx

```diff
@@ -57,19 +57,24 @@ describe('the EdgeResult component', () => {
     expect(wrapper.find(ResultIcon).exists()).toBe(false);
   });
 
-  [{
-    codecPreferences: [Opus],
-    label: 'Ashburn (Opus)',
-  },{
-    codecPreferences: [PCMU],
-    label: 'Ashburn (PCMU)',
-  },{
-    codecPreferences: [Opus, PCMU],
-    label: 'Ashburn (Opus, PCMU)',
-  },{
-    codecPreferences: [PCMU, Opus],
-    label: 'Ashburn (PCMU, Opus)',
-  }].forEach(test => {
+  [
```

Comment:
@charliesantos Not a big deal, but I'm not sure if `prettier` is working right for you. These files should be automatically be formatted on each commit by the husky git hooks: https://github.com/twilio/rtc-diagnostics-react-app/blob/main/package.json#L86

Not sure why this would be. This should all be set up just by running `npm install`.